src/main/java/core/basesyntax/service/impl/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitInventory;
+import core.basesyntax.service.ReportGenerator;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String REPORT_HEADER = "fruit,quantity" + System.lineSeparator();
+
+    @Override
+    public void generate(String path) {
```

Comment:
It is better to use a FileWriter or BufferedWriter instead of using File and Files for writing to a csv file.